nav2_costmap_2d/src/costmap_2d_ros.cpp

```diff
@@ -383,11 +418,15 @@ void Costmap2DROS::start()
   // block until the costmap is re-initialized.. meaning one update cycle has run
   rclcpp::Rate r(100.0);
   while (rclcpp::ok() && !initialized_) {
+    RCLCPP_DEBUG(get_logger(), "Sleeping, waiting for initialized_");
     r.sleep();
   }
+
+  RCLCPP_INFO(get_logger(), "returning");
```

Comment:
This logging is pretty non-descriptive and probably unnecessary. If we keep it it should contain more information.

Reply:
Sure. Deleted.